staging_service/autodetect/GenerateMappings.py

```diff
@@ -23,99 +22,33 @@
-    },
-    fastq_reads_interleaved_id: {
-        "title": "FastQ Reads Interleaved",
-        "app": "kb_uploadmethods/import_fastq_interleaved_as_reads_from_staging",
-        "output_type": ["KBaseFile.SingleEndLibrary", "KBaseFile.PairedEndLibrary"],
-    },
-    fastq_reads_noninterleaved_id: {
-        "title": "FastQ Reads NonInterleaved",
-        "app": "kb_uploadmethods/import_fastq_noninterleaved_as_reads_from_staging",
-        "output_type": ["KBaseFile.SingleEndLibrary", "KBaseFile.PairedEndLibrary"],
-    },
-    assembly_id: {
-        "title": "Assembly",
-        "app": "kb_uploadmethods/import_fasta_as_assembly_from_staging",
-        "output_type": ["KBaseGenomeAnnotations.Assembly"],
-    },
-    gff_genome_id: {
-        "title": "GFF/FASTA Genome",
-        "app": "kb_uploadmethods/import_gff_fasta_as_genome_from_staging",
-        "output_type": ["KBaseGenomes.Genome"],
-    },
-    gff_metagenome_id: {
-        "title": "GFF/FASTA MetaGenome",
-        "app": "kb_uploadmethods/import_gff_fasta_as_metagenome_from_staging",
-        "output_type": ["KBaseMetagenomes.AnnotatedMetagenomeAssembly"],
-    },
-    genbank_genome_id: {
-        "title": "Genbank Genome",
-        "app": "kb_uploadmethods/import_genbank_as_genome_from_staging",
-        "output_type": ["KBaseGenomes.Genome"],
-    },
-    decompress_id: {
-        "title": "Decompress/Unpack",
-        "app": "kb_uploadmethods/unpack_staging_file",
-        "output_type": [None],
-    },
-    sample_set_id: {
-        "id": sample_set_id,
-        "title": "Samples",
-        "app": "sample_uploader/import_samples",
-        "output_type": ["KBaseSets.SampleSet"],
-    },
-    media_id: {
-        "title": "Media",
-        "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
-        "output_type": ["KBaseBiochem.Media"],
-    },
-    expression_matrix_id: {
-        "title": "Expression Matrix",
-        "app": "kb_uploadmethods/import_tsv_as_expression_matrix_from_staging",
-        "output_type": ["KBaseFeatureValues.ExpressionMatrix"],
-    },
-    metabolic_annotations_id: {
-        "title": "Metabolic Annotations",
-        "app": "MergeMetabolicAnnotations/import_annotations",
-        "output_type": ["KBaseGenomes.Genome"],
-    },
-    metabolic_annotations_bulk_id: {
-        "title": "Bulk Metabolic Annotations",
-        "app": "MergeMetabolicAnnotations/import_bulk_annotations",
-        "output_type": ["KBaseGenomes.Genome"],
-    },
-    fba_model_id: {
-        "title": "FBA Model",
-        "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
-        "output_type": ["KBaseFBA.FBAModel"],
-    },
-    phenotype_set_id: {
-        "title": "Phenotype Set",
-        "app": "kb_uploadmethods/import_tsv_as_phenotype_set_from_staging",
-        "output_type": ["KBasePhenotypes.PhenotypeSet"],
-    },
-    escher_map_id: {
-        "title": "EscherMap",
-        "app": "kb_uploadmethods/import_eschermap_from_staging",
```

Comment:
All of the output object types are now gone? Sad

Reply:
Honestly I'm relieved, since if an app is re-registered with catalog/NMS and the types change, the staging service will be out of sync. Better to force the client programmer to pull from the source of truth